### PR TITLE
[MIBM-253][JR] GTM

### PR DESCRIPTION
**Pull request by jordanrowe**

### CODE REVIEW

**Review by Leeroyc720 — APPROVED** [review]

File: conf/application.conf [hunk 1]
```diff
@@ -45,7 +45,7 @@ play.modules.enabled += "play.modules.reactivemongo.ReactiveMongoHmrcModule"
 
 play.modules.enabled += "uk.gov.hmrc.merchandiseinbaggage.config.MerchandiseInBaggageFrontendConfigModule"
 
-play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com data:"
```
Comment (sureshhmrc) [inline conf/application.conf:48]: `localhost:12345` ? is this needed

File: conf/application.conf [hunk 2]
```diff
@@ -130,6 +130,10 @@ google-analytics {
   host = auto
 }
 
+tracking-consent-frontend {
+  gtm.container = "a"
```
Comment (sureshhmrc) [inline conf/application.conf:134]: are you creating app-config PRs for adding actual tracking code ?  or is the tracking code not ready yet ?

File: conf/application.conf (hunk 1 above)
Comment (jordanrowe) [reply conf/application.conf:48]: yeah, check this page for more info: https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=PLATUI&title=Integrating+with+tracking+consent+-+a+guide+for+service+teams

File: conf/application.conf (hunk 2 above)
Comment (jordanrowe) [reply conf/application.conf:134]: it's just a container a,b,c,d,e or f and ticket says to use a, not sure how it actually works behind the scenes

**Pull request merged**